### PR TITLE
[Chore] More tezos modules options

### PR DESCRIPTION
**Pull request by rvem**
## Description

This PR adds a few options to the existing NixOS modules with background Octez daemons.

## Related issue(s)

https://issues.serokell.io/issue/TM-645

#### Related changes (conditional)

- [x] I checked whether I should update the [README](/serokell/tezos-packaging/tree/master/README.md)

- [x] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: nix/modules/tezos-baker.nix [hunk 1]
```diff
@@ -27,6 +37,12 @@ let
         '';
       };
 
+      liquidityBakingToggleVote = mkOption {
+        type = types.enum [ "on" "of" "pass" ];
+        default = "pass";
+        description = "Vote to continue (option "on") or end (option "off") the liquidity baking subsidy. Or choose to pass (option "pass")";
```
Comment (ixahedron) [inline nix/modules/tezos-baker.nix:43]: Does nix handle the quotes-inside-quotes correctly? I'd use multi-line string for description as in the previous option, for consistency and to keep everything ~80 chars wide, but at the minimum I suspect you'd have to escape them.

File: nix/modules/tezos-baker.nix [hunk 2]
```diff
@@ -12,13 +12,23 @@ let
...
   instanceOptions = types.submodule ( {...} : {
     options = common.daemonOptions // {
 
       enable = mkEnableOption "Tezos baker service";
 
+      bakerSecretKey = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = ''
+          Secret key stat will be imported as 'bakerAccountAlias' before
```
Comment (ixahedron) [inline nix/modules/tezos-baker.nix:27]: > stat

:thinking: 

```suggestion
          If provided, secret key will be imported for 'bakerAccountAlias' before
```

**Review by ixahedron — APPROVED** [review]
LGTM aside from a couple nitpicks

File: nix/modules/tezos-baker.nix (hunk 2 above)
Comment (rvem) [reply nix/modules/tezos-baker.nix:27]: Hmm, not sure what I meant here 🤔

**Pull request merged**